### PR TITLE
updated readme, added docker functionality

### PR DESCRIPTION
**Pull request by akram-rameez**
I looked at @phated 's comment, and added a return, to return the stream. I'm a little new to gulp, so are the changes _cool_??

updated the readme, and tried to keep it as **pet neutral** as possible.

While on the subject on staying hip and cool, I added docker functionality, so you can run the app in container out of the box :stuck_out_tongue:

### CODE REVIEW

**Comment by phated** [comment]
Yeah, the return is proper.

**Comment by akram-rameez** [comment]
@phated awesome :smile:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+- We all :heart: pets
+- We all :heart: orange
+- Hence we all :heart: Orange Pets
+
+## Running Locally
+1. Clone the git repository
+2. Make sure you have [node](https://nodejs.org/) installed on your system by running `npm -v`, which should log the current version of *node* installed.
+3. Run `npm install`.
+4. Run `npm start` to run the app.
+5. Run `npm build` to build a production ready version of the app.
```
Comment (lynnaloo) [inline README.md:15]: production ready 😂 lolz

File: README.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+## Why Oran.js?:
+
+- We all :heart: pets
+- We all :heart: orange
+- Hence we all :heart: Orange Pets
+
+## Running Locally
+1. Clone the git repository
+2. Make sure you have [node](https://nodejs.org/) installed on your system by running `npm -v`, which should log the current version of *node* installed.
+3. Run `npm install`.
```
Comment (lynnaloo) [inline README.md]: you can `npm i` here as a shortcut

**Comment by lynnaloo** [comment]
Thanks for the code review @phated 😉

**Comment by akram-rameez** [comment]
lets merge this bad boy?

**Pull request merged**